README.md

```diff
@@ -1,104 +1,210 @@
 
-`qhub` is installed as a command line application in Python. It requires you to choose your the cloud provider you desire. once you've decided on a provider `qhub` will walk you through the following steps to configure your deployment:
+## QHub Cloud
+The cloud version of QHub is built using [Terraform](https://www.terraform.io/), [Helm](https://helm.sh/), and 
+[GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions).
 
-1. initialize
-2. render
-3. deploy
+### Cloud Providers
+QHub offers out-of-the-box support for [Digital Ocean], Amazon [AWS] and [GCP] (Google Cloud Provider). Although by 
+default, as long as the chosen provider offers Kubernetes services and has a Terraform module, it is possible to 
+configure QHub to deploy your project.
 
-these steps realized using the `qhub` CLI.
 
-### initialize configurations
+![image](docs/images/brand-diagram.png "architecture diagram")
 
-        qhub init do
-        qhub init aws
-        qhub init gcp
+For more details, check out the release [blog post](https://www.quansight.com/post/announcing-qhub).
 
-The `qhub init` command will generate configuration files for that service. The configutation files can be tailored to the needs of your organization. Each file specifies general project information, security, infrastructure settings, computational resource profiles and data science environments. See documentation on modifying your configuration file for all of the cloud providers: [Configuration File](https://github.com/Quansight/qhub/blob/master/docs/docs/aws/configuration.md) 
+## Usage
+### Installation pre-requisites
+* QHub is supported by macOS and Linux operating systems (Windows is **NOT** currently supported);
+* We recommend the adoption of virtual environments (`conda`, `pipenv` or `venv`) for successful usage. 
+
+### Install
+QHub's installation can be performed by using:
+* `conda`:
+  ```bash
+  conda install -c conda-forge qhub
+  ```
+  
+* or `pip` (instead):
+    ```bash
+    pip install qhub
+    ```  
+Once finished, you can check QHub's version (and additional CLI args) by typing:
+```bash
+qhub --help
+```
+If successful, the CLI output will be similar to the following:
 
-The configuration file is your user interface into deploying and scaling your data science environment. Each change triggers [Github Action] that will seamlessly update your infrastructure.
+![help_cli_output.png](docs/images/help_cli_output.png)
 
-![](docs/images/brand-diagram.png "architecture diagram")
+### Set up initialization
+#### 1. Cloud Provider
+The first required step is to **choose a Cloud Provider to host the project deployment**. The cloud installation is based
+on Kubernetes, but knowledge of Kubernetes is **NOT** required. By default, QHub uses DigitalOcean, AWS and GCP. If you 
+wish to use a different provider, make sure they support Kubernetes and Terraform and follow the suitable documentation
+to configure it.
```

Comment:
From Kim:
>we shout out that kubernetes experience is not required, as a non-dev-ops person, I don't have terraform experience either. Below here, Terraform is mentioned several times. Users need to know "what is terraform" and "is it something I need to learn about?". I think those can be addressed in this paragraph.

Reply:
Yup! That is a great point! Will address it briefly here but also think we should have a longer more detailed explanation on those at the FAQ.

Thanks @kim and tyler :)